README.md

```diff
@@ -34,24 +34,24 @@ npm install -g truffle
 mkdir myproject && cd myproject
 truffle init
 ```
-To install the daostack-arc library, run:
+To install the @daostack/arc library, run:
```

Comment:
Should it be : 

> To instal arc package ,run : 
```sh
 npm init
 npm install @daostack/arc
  ```		  ```

Reply:
You can do `npm install` without `npm init`.